server/backends/chunkstore/chunkstore.go

```diff
@@ -251,6 +251,10 @@ func (r *chunkstoreReader) shiftToFront(p []byte, bytesRead *int) {
 }
 
 func (r *chunkstoreReader) Read(p []byte) (int, error) {
+	return r.read(p)
+}
+
+func (r *chunkstoreReader) read(p []byte) (int, error) {
```

Comment:
nit: seems like this diff is no longer needed?

Reply:
oops, yeah.

```suggestion
```